packages/select/src/FilterMultiSelect/components/ListBoxSection/ListBoxSection.tsx

```diff
@@ -1,24 +1,54 @@
-import React from "react"
+import React, { ReactNode } from "react"
+import { v4 } from "uuid"
 import { MultiSelectItem } from "../../../types"
 import styles from "./ListBoxSection.module.scss"
-
 export interface ListBoxSectionProps {
   items: MultiSelectItem[]
   /**
    * Becomes an aria-label on the section, informing
    * unsighted users
    */
   sectionName: string
+  /** If provided, will override the aria-label for this group */
+  sectionHeader?: ReactNode
   children: (item: MultiSelectItem) => React.ReactNode
 }
 
 export const ListBoxSection = ({
   items,
   sectionName,
   children,
-}: ListBoxSectionProps): JSX.Element => (
-  <div aria-label={sectionName} className={styles.listBoxSection}>
-    {Array.from(items).map(node => node != undefined && children(node))}
-  </div>
-)
+  sectionHeader,
+}: ListBoxSectionProps): JSX.Element => {
+  const listSectionId = sectionHeader ? v4() : undefined
```

Comment:
This will need to be stored in state to avoid regen on rerender

Reply:
Actually now I'm thinking about it, do think we should use the useId hook from react?

Reply:
❌  We can't use React 18 features yet